packages/babel-plugin-transform-object-rest-spread/src/index.js

```diff
@@ -175,11 +175,11 @@ export default function ({ types: t }) {
           let nodeWithoutSpread = t.clone(path.node);
           nodeWithoutSpread.right = ref;
           nodes.push(t.expressionStatement(nodeWithoutSpread));
-          nodes.push(t.assignmentExpression(
+          nodes.push(t.toStatement(t.assignmentExpression(
```

Comment:
Want to be able to fix this automatically somehow - linting rule or babel should add it automatically?